Resources/Prototypes/_RMC14/Entities/Mobs/Xeno/rouny.yml

```
@@ -0,0 +1,20 @@
+﻿- type: entity
+  parent:
+  - CMXenoRunner
+  id: RMCXenoRouny
+  name: Rouny
+  suffix: RMC14
+  description: thas a fuckin rouny
+  - type: Sprite
```

This is missing `components:`
```suggestion
  description: thas a fuckin rouny
  components:
  - type: Sprite
```

---

i knew i missed something grahhh

---

i swear i got that components: added when I re-did the PR on steam deck, guess I forgot to re-commit to master on kdo-rmc14 lmao